src/app/components/tx-item.tsx

```diff
@@ -31,18 +31,39 @@ import {
 import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
 import { useFungibleTokenMetadata } from '@app/query/tokens/fungible-token-metadata.hook';
 import { pullContractIdFromIdentity } from '@app/common/utils';
+import { FiInfo } from 'react-icons/fi';
 
 type Tx = MempoolTransaction | Transaction;
+const pendingWaitingMessage =
+  'This transaction is waiting to be confirmed. Please be patient as this may take between a few minutes to a few hours.';
 
 const Status = ({ transaction, ...rest }: { transaction: Tx } & BoxProps) => {
   const isPending = isPendingTx(transaction as MempoolTransaction);
   const isFailed = !isPending && transaction.tx_status !== 'success';
   return isFailed || isPending ? (
     <Box {...rest}>
       {isPending && (
-        <Caption variant="c2" color={color('feedback-alert')}>
-          Pending
-        </Caption>
+        <Flex>
+          <Caption variant="c2" color={color('feedback-alert')}>
+            Pending
+          </Caption>
+          <Tooltip
+            label={pendingWaitingMessage}
+            labelProps={{ bg: '#e2e2e2', padding: '5px', borderRadius: '5px' }}
+            placement="bottom"
+          >
+            <Stack>
+              <Box
+                _hover={{ cursor: 'pointer' }}
+                as={FiInfo}
+                color={color('text-caption')}
+                ml={1}
+                mt={-1}
+                size="14px"
+              />
+            </Stack>
+          </Tooltip>
+        </Flex>
```

Comment:
Could this be a new component?

`{isPending && <PendingLabel />}` would be cleaner